Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:8-jdk-alpine
+WORKDIR /
+COPY conter-0.0.1-SNAPSHOT.jar app.jar
```

Comment:
Once #6 is merged, this will have to change to `counter`; the version is already incorrect since #5 was merged.